doc/whats_new/upcoming_changes/sklearn.metrics/27412.fix.rst

```diff
@@ -1,3 +1,3 @@
-- :func:`metrics.roc_auc_score` will now correctly return 0.0 and
+- :func:`metrics.roc_auc_score` will now correctly return np.nan and
   warn user if only one class is present in the labels.
   By :user:`Gleb Levitski <glevv>`
```

Comment:
```suggestion
  By :user:`Gleb Levitski <glevv>` and :user:`Janez Demšar <janzed>`.
```

Reply:
Or do we need to edit this changelog and create a second one with this changelog text and this PR's number to have both in the changelog? Would the `By user...` line then be ignored in the text comparison and merge of the chagenlog entries?

cc @lesteve @glemaitre